aten/src/ATen/native/xnnpack/Convolution.cpp

```diff
@@ -170,7 +170,7 @@ Tensor run(
       "XNNPACK Convolution not usable! "
       "Reason: The provided input tensor is either invalid or unsupported by XNNPACK.");
 
-  Tensor output = empty_with_tail_padding(
+  Tensor output_nhwc = empty_with_tail_padding(
```

Comment:
These changes should probably be a separate diff.

Reply:
Yes please. Can you remove these changes? Else I or you will run into some merge conflict.

Reply:
OK, no worries, will revert.